src/main.rs

```diff
@@ -116,3 +122,18 @@ async fn main() {
         exit(1);
     }
 }
+
+/// Expand ~ and any environment variables in the given path.
+///
+/// # Arguments
+///
+/// * `path` - Path to expand.
+///
+/// # Returns
+///
+/// Path with ~ and environment variables expanded.
+fn expand_path(path: &Path) -> PathBuf {
```

Comment:
I noticed that we weren't translating `~/.evergreen.yml`, so I went ahead and added some code to fix that.